iron-overlay-manager.html

```diff
@@ -22,6 +23,8 @@
 
     _backdrops: [],
 
+    _backdropElement: document.createElement('iron-overlay-backdrop'),
```

Comment:
Oh bummer...
Well, thinking of it again, it makes sense for `iron-overlay-behavior` to do the import, after all he's the one that uses the backdropElement's methods. Sorry for the confusion, my previous suggestion introduced this issue :disappointed: